### PR TITLE
Revert #132806, #132736, #132539, #132487

### PR DESCRIPTION
**Pull request by ezyang**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #133570

This reverts commit 25df063f044202899ab92d6f3d77950af5de482f.
This reverts commit de00c7958301ce81b9716bdef5731ed40d4d14ca.
This reverts commit 419b76c4ac80c8b1c95120cd52db622333a3a688.
This reverts commit bc57d5b6ff8725bbe93f0e67db72459720c750cf.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

Differential Revision: [D61335013](https://our.internmc.facebook.com/intern/diff/D61335013)

### CODE REVIEW

**Review by albanD — APPROVED** [review]
Stampy accept to fix!

**Comment by ezyang** [comment]
@ezyang has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D61335013).

**Comment by ezyang** [comment]
@ezyang has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D61335013).

**Comment by ezyang** [comment]
@ezyang has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D61335013).

**Comment by ezyang** [comment]
@ezyang has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D61335013).

**Review by jansel — APPROVED** [review]

File: test/inductor/test_cpu_select_algorithm.py [hunk 1]
```diff
@@ -739,6 +730,7 @@ def forward(self, x, other, other2):
 
     @inductor_config.patch({"freezing": True})
     @patches
+    @torch._dynamo.config.patch(inline_inbuilt_nn_modules=True)
```
Comment (anijain2305) [inline test/inductor/test_cpu_select_algorithm.py:733]: I think the current failures could be resolved by removing this line.

**Comment by ezyang** [comment]
@ezyang has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D61335013).

**Review by anijain2305 — APPROVED** [review]

**Pull request closed without merging**